### PR TITLE
Update Badger Wallets and adds latest Votium Data

### PR DESCRIPTION
**Pull request by sajanrajdev**
Relates to #1169

### CODE REVIEW

File: helpers/addresses.py [hunk 1]
```diff
@@ -787,19 +787,20 @@
...
         "guardian_backups": {
             "defender1": "0x4666e94300429C6d3340449bC828E4218F360672",
             "defender2": "0x211b82242076792A07C7554A5B968F0DE4414938",
-            "ops_executor7": "0xaaE2051c2f74920C6662EF5C9B0d602C40D36DF4",
         },
         "_deprecated": {
             "ops_deployer6": "0x96AC69183216074dd8CFA7A380e873380445EaDc",
             "ops_deployer7": "0x7140B5254d80154f9Fc5F86054efB210f3a1e2c6",
             "ops_deployer9": "0xd49177c4cF4b8cCdf58475c9a155C4CD8b9F3F31",
+            "ops_deployer8": "0x9082b0dD7A72c328833e6461965C9E91Cf59a960",
```
Comment (gosuto-inzasheru) [inline helpers/addresses.py:802]: tambien

File: helpers/addresses.py [hunk 2]
```diff
@@ -127,9 +124,12 @@
             "ops_deployer6_v2": "0x96AC69183216074dd8CFA7A380e873380445EaDc",
             "ops_deployer7": "0x7140B5254d80154f9Fc5F86054efB210f3a1e2c6",
             "ops_deployer9": "0xd49177c4cF4b8cCdf58475c9a155C4CD8b9F3F31",
+            "ops_deployer8": "0x9082b0dD7A72c328833e6461965C9E91Cf59a960",
```
Comment (gosuto-inzasheru) [inline helpers/addresses.py:127]: wrong order this one 😝

**Review by gosuto-inzasheru — COMMENTED** [review]
can confirm deployer8, executor7 and executor13

File: helpers/addresses.py (hunk 2 above)
Comment (sajanrajdev) [reply helpers/addresses.py:126]: Oups, good catch. Fixed.

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]
confirm onboarding of exec16 and offboarding addrs ✅

**Pull request merged**